### PR TITLE
Enable publishing camera image maintaining publishing people pose 

### PR DESCRIPTION
**Pull request by iory**
# What is this?

At current, unitree don't publish camera image.
This PR enables publishing camera images maintaining publishing people's pose.
live_human_pose.py publishes people pose from gstreamer images. live_human_pose.py has been modified to output images via MQTT.
Also changed to start live_human_pose.py on jetson nano2gb (192.168.123.13).

## Camera images

https://user-images.githubusercontent.com/4690682/183285233-d697c566-384b-4a38-a714-6f262ea7b4ba.mp4

## tar.gz

https://drive.google.com/drive/folders/1Ul3rXMXogUEvX6_wZunXwxy33zfHMzeX?usp=sharing

cc: @tkmtnt7000

### CODE REVIEW

File: jsk_unitree_robot/cross/install.sh [hunk 1]
```diff
@@ -142,8 +145,13 @@ if [[ "${TARGET_DIRECTORY}" == "User" ]]; then
     sshpass -p 123 scp ${TARGET_MACHINE}_${TARGET_DIRECTORY}/src/jsk_robot/jsk_unitree_robot/jsk_unitree_startup/scripts/publish_human_pose.diff unitree@${cuda_ip}:/tmp/publish_human_pose.diff
     sshpass -p 123 ssh -t unitree@${cuda_ip} bash -c 'ls; OUT="$(patch -p0 --backup --forward /home/unitree/Unitree/autostart/imageai/mLComSystemFrame/pyScripts/live_human_pose.py < /tmp/publish_human_pose.diff | tee /dev/tty)" || echo "${OUT}" | grep "Skipping patch" -q || (echo "$OUT" && false);'
 
-    if [[ ${TYPE} == "Air" ]] ; then
-        sshpass -p 123 ssh -t unitree@${cuda_ip} "sed -i 's/192.168.123.15/192.168.123.13/g' /home/unitree/Unitree/autostart/imageai/mLComSystemFrame/pyScripts/live_human_pose.py"
+    # launch live_human_pose.py on jetson nano (192.168.123.13)
```
Comment (k-okada) [inline jsk_unitree_robot/cross/install.sh:148]: why you changed to start publish_human_pose from 192.168.123.15 to 192.168.123.13? To have same setting for both Air/Pro?

File: jsk_unitree_robot/jsk_unitree_startup/autostart/imageai.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```
Comment (k-okada) [inline jsk_unitree_robot/jsk_unitree_startup/autostart/imageai.sh:1]: Is this a copy from unitree robot? If so please use install.sh to download and copy it.Do we need to install this for Air?

**Review by k-okada — COMMENTED** [review]
please create table to show which CPU runs which nodes
somethig like ...
|CPU| Air | Pro |
| ------ | ------ | ----- |
|161 (raspberryPi ...) | sound_play_node.launch, rwt_app_chooser.launch, rosserial_node.launch, sample_respekaer.launch` | (same as Air) |
| 13 ( jetson nano, 32G?) |-| jsk_unitree_startup.launch| |
| 14 ( jetson nano, 32G?) |-| | 
| 15 ( jetson nano, 32G?) |  jsk_unitree_startup.launch| |

**Comment by k-okada** [comment]
@iory please include https://github.com/k-okada/jsk_robot/pull/60 so that it send images while lead_teleop

note that I will merge https://github.com/jsk-ros-pkg/jsk_robot/pull/1558, in a few days

**Comment by iory** [comment]
OK. I'll tackle it.

File: jsk_unitree_robot/cross/install.sh (hunk 1 above)
Comment (iory) [reply jsk_unitree_robot/cross/install.sh:148]: Sorry. This change is no longer necessary.
This was to publish the image with ROS.

Instead, I used mqtt to publish.
https://github.com/k-okada/jsk_robot/pull/69/commits/9b2b375d5dab45acad4f4a914dcf37d1d44c2070

File: jsk_unitree_robot/jsk_unitree_startup/autostart/imageai.sh (hunk 2 above)
Comment (iory) [reply jsk_unitree_robot/jsk_unitree_startup/autostart/imageai.sh:1]: Sorry. This change is no longer necessary.
This was to publish the image with ROS.

Instead, I used mqtt to publish.
https://github.com/k-okada/jsk_robot/pull/69/commits/9b2b375d5dab45acad4f4a914dcf37d1d44c2070

**Comment by iory** [comment]
Move to https://github.com/k-okada/jsk_robot/pull/69

**Pull request closed without merging**